src/npcmove.cpp

```diff
@@ -1364,55 +1392,54 @@ void npc::move_pause()
 
 void npc::find_item()
 {
+fetching_item = false;
```

Comment:
The function, it does (nearly) nothing. Is this intended? It looks like this prevents NPCs from picking up *any* item at all (as they never find one).